src/utils/helpers/i18n/i18n.js

```diff
@@ -58,15 +58,6 @@ const I18nHelper = {
       groupSize: 3,
     };
 
-    // valueToFormat is "en" formatted.
```

Comment:
Please restore this, this could be consumed by other projects. Looks like we don't have a unit test for this functionality. Please can you add one.

Reply:
Unfortunately it is not in a `__internal__` folder so we consider it part of the public API. This helper is available as 
```js
import {formatDecimal} from "carbon-react/lib/utils/helpers/i18n";
```

In the short term we plan to remove these helpers so I think it's OK to remove it like you have as long as it's documented in the commit as a breaking change. 

I think the commit message should be something like this


```
feat(decimal): add locale support

BREAKING CHANGE: The decimal component no longer uses `i18n-js` for decimal formatting, you should use the `locale` prop instead. 

BREAKING CHANGE: `formatDecimal` no longer supports the `round` option first introduced in v73.0.0

closes FE-3500
```